src/pykeen/nn/init.py

```diff
@@ -494,7 +494,7 @@ def __init__(
 #: - :func:`pykeen.nn.init.xavier_normal_norm_`
 #:
 #: as well as initializers from :mod:`torch.nn.init`.
-initializer_resolver: FunctionResolver[Initializer] = FunctionResolver(
+initializer_resolver: FunctionResolver[str, LabeledTriples] = FunctionResolver(
```

Comment:
almost! you need to put `str` in a list, since this is a ParamSpec

```suggestion
initializer_resolver: FunctionResolver[[str], LabeledTriples] = FunctionResolver(
```